luaui/Widgets/cmd_guard_remove.lua

```diff
@@ -51,12 +58,24 @@ function widget:UnitCommand(unitID, unitDefID, unitTeam, cmdId, cmdParams, cmdOp
 		end
 	end
 
-	doCommandRemove = false
 	return false
 end
 
-function widget:KeyPress(key, modifier, isRepeat)
-	if not isRepeat and (key == KEYSYMS.LSHIFT or key == KEYSYMS.RSHIFT) then
-		doCommandRemove = true
+function widget:Update(dt)
+	updateTime = updateTime + dt
+	if updateTime < 0.25 then
+		return
+	end
+
```

Comment:
Don't you need to set updateTime to 0 once it passes .25?

Reply:
Annd that's why I need my code reviewed 😅